sdk/eventhubs/azure-messaging-eventhubs/src/main/java/com/azure/messaging/eventhubs/EventHubConsumerAsyncClient.java

```diff
@@ -163,52 +136,122 @@ public String getConsumerGroup() {
      * events in the partition event stream.
      *
      * @param partitionId The unique identifier of a partition associated with the Event Hub.
+     *
      * @return The set of information for the requested partition under the Event Hub this client is associated with.
      */
     @ServiceMethod(returns = ReturnType.SINGLE)
     public Mono<PartitionProperties> getPartitionProperties(String partitionId) {
+        if (Objects.isNull(partitionId)) {
+            return monoError(logger, new NullPointerException("'partitionId' cannot be null."));
+        } else if (partitionId.isEmpty()) {
+            return monoError(logger, new IllegalArgumentException("'partitionId' cannot be an empty string."));
+        }
+
         return connection.getManagementNode().flatMap(node -> node.getPartitionProperties(partitionId));
     }
 
+
+    /**
+     * Begin consuming events from a single partition starting at {@code startingPosition}.
+     *
+     * @param partitionId Identifier of the partition to read events from.
+     * @param startingPosition Position within the Event Hub partition to begin consuming events.
+     *
+     * @return A stream of events for this partition. If a stream for the events was opened before, the same position
```

Comment:
Why would this use the same position? If I make two calls `receiveFromPartition("0", EventPosition.earliest())` and `receiveFromPartition("0", EventPosition.latest())` - what would this result in?

Reply:
This doc was a carry-over from the old implementation where there could only be one receiver. The docs have been fixed.